apps/wire-webapp/.lintstagedrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "*": "prettier --ignore-unknown --write",
+  "*.less": ["stylelint --fix"],
+  "*.{js,jsx,ts,tsx}": ["eslint --fix"]
+}
```

Comment:
This is the way to go for a monorepo structure (see https://github.com/lint-staged/lint-staged?tab=readme-ov-file#how-to-use-lint-staged-in-a-multi-package-monorepo)